ui/app/src/views/ViewDashboard.tsx

```diff
@@ -30,9 +30,11 @@ function ViewDashboard() {
   }
 
   return (
-    <DashboardView dashboardResource={dashboard}>
-      <Footer />
-    </DashboardView>
+    <DashboardProvider initialState={{ dashboardSpec: dashboard.spec }}>
```

Comment:
Do you think this should actually go inside `DashboardView` from the `dashboards` package? Not clear to me really, but thought I should ask since the other providers are all in there.

Reply:
I had to put the `DashboardProvider` here because `DashboardView` needs it to access the store.

I can move other providers to `ViewDashboard` though 😜

Reply:
So when integrating Perses as a framework into external apps, we import the [`Dashboard`](https://github.com/perses/perses/blob/164c34505d55867283f10acc760e154604b7ff5a/ui/dashboards/src/components/Dashboard.tsx#L26) component from the `dashboards` package then wire up the Providers by hand. Is our thought that `ViewDashboard` can be seen as a "helper preset component" to do this for you. If so, I can see this Provider moving into that component.

Reply:
I don't think we really talked about that yet, but you make a good point. 

I can move the logic that needs access to the store into a separate component so that I can move `DashboardProvider` to `ViewDashboard`